Wrappers/Python/ccpi/viewer/ui/VolumeRenderSettingsDialog.py

```diff
@@ -0,0 +1,287 @@
+
+    def _setUpWindowingMax(self):
+        windowing_slider_max = UISliderWidget.UISliderWidget(0.0, 100.0)
+
+        self.addWidget(windowing_slider_max, "Windowing Maximum (%):", "windowing_slider_max")
+        self.formWidget.widgets["windowing_slider_max_label"].setToolTip(
+            TOOLTIPS_VOLUME_RENDER_SETTINGS["windowing_slider_max"])
+
+        self.getWidget("windowing_slider_max").setValue(self.default_slider_values["windowing_max"])
+        self.getWidget("windowing_slider_max").slider.valueChanged.connect(self.updateVolumeOpacityMax)
+        self.getWidget("windowing_slider_max").line_edit.editingFinished.connect(self.updateVolumeOpacityMax)
+
+    def _setUpOpacityMapping(self):
+        opacity_mapping = QtWidgets.QComboBox(self.groupBox)
+
+        opacity_mapping.addItems(["Scalar", "Gradient"])
+        self.addWidget(opacity_mapping, "Opacity Mapping:", "opacity_mapping")
+        self.formWidget.widgets["opacity_mapping_label"].setToolTip(TOOLTIPS_VOLUME_RENDER_SETTINGS["opacity_mapping"])
+
+        self.getWidget("opacity_mapping").currentIndexChanged.connect(self.setOpacityMapping)
+
+    def _setUpColourScheme(self):
+        colour_scheme = QtWidgets.QComboBox(self.groupBox)
+
+        colour_scheme.addItems(color_scheme_list())
+        self.addWidget(colour_scheme, "Colour Scheme:", "colour_scheme")
+        self.formWidget.widgets["colour_scheme_label"].setToolTip(TOOLTIPS_VOLUME_RENDER_SETTINGS["colour_scheme"])
+
+        self.getWidget("colour_scheme").currentIndexChanged.connect(self.setColourScheme)
+
+    def _setUpVolumeClipping(self):
+        volume_clipping = QtWidgets.QCheckBox("Volume Clipping", self.groupBox)
+        volume_clipping_reset = QtWidgets.QPushButton("Reset Volume Clipping", self.groupBox)
+
+        self.addWidget(volume_clipping, "", "volume_clipping")
+        self.addWidget(volume_clipping_reset, "", "volume_clipping_reset")
+        self.formWidget.widgets["volume_clipping_field"].setToolTip(TOOLTIPS_VOLUME_RENDER_SETTINGS["volume_clipping"])
+        self.formWidget.widgets["volume_clipping_reset_field"].setToolTip(
+            TOOLTIPS_VOLUME_RENDER_SETTINGS["volume_clipping_reset"])
+
+        self.getWidget("volume_clipping").stateChanged.connect(self.viewer.style.ToggleVolumeClipping)
+        self.getWidget("volume_clipping_reset").clicked.connect(self.resetVolumeClipping)
+
+    def _setUpColourRangeMin(self):
+        colour_range_slider_min = UISliderWidget.UISliderWidget(0.0, 100.0)
+
+        self.addWidget(colour_range_slider_min, "Colour Range Minimum (%):", "colour_range_slider_min")
+        self.formWidget.widgets["colour_range_slider_min_label"].setToolTip(
+            TOOLTIPS_VOLUME_RENDER_SETTINGS["colour_range_slider_min"])
+
+        self.getWidget("colour_range_slider_min").setValue(self.default_slider_values["colour_range_min"])
+        self.getWidget("colour_range_slider_min").slider.valueChanged.connect(self.updateColourRangeMin)
```

Comment:
```suggestion
        self.getWidget("colour_range_slider_min").slider.sliderReleased.connect(self.updateColourRangeMin)
```
We should connect [`sliderReleased`](https://doc.qt.io/qt-6/qabstractslider.html#sliderReleased) instead of `valueChanged` in all the sliders.